### PR TITLE
Clean up paradox output

### PR DESCRIPTION
**Pull request by raboof**
Fix some validation problems

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -250,7 +250,7 @@ lazy val docs = akkaModule("akka-docs")
       "extref.akka.http.base_url" -> "https://doc.akka.io/docs/akka-http/current/%s",
       "extref.wikipedia.base_url" -> "https://en.wikipedia.org/wiki/%s",
       "extref.github.base_url" -> (GitHub.url(version.value) + "/%s"), // for links to our sources
-      "extref.samples.base_url" -> "https://developer.lightbend.com/start/?group=akka&project=%s",
+      "extref.samples.base_url" -> "https://developer.lightbend.com/start/?group=akka&amp;project=%s",
```
Comment (johanandren) [inline build.sbt]: You double checked (or know for sure) that this works I assume?

**Review by johanandren — APPROVED** [review]
LGTM

File: build.sbt (hunk 1 above)
Comment (johanandren) [reply build.sbt]: I mean that there isn't any further escaping or so

Comment (raboof) [reply build.sbt]: yes, tested in firefox, no, doesn't seem to be any additional escaping elsewhere.

File: akka-docs/src/main/paradox/security/2017-02-10-java-serialization.md [hunk 2]
```diff
@@ -27,7 +27,7 @@ Please subscribe to the [akka-security](https://groups.google.com/forum/#!forum/
 
 ### Severity
 
-The [CVSS](https://en.wikipedia.org/wiki/CVSS) score of this vulnerability is 6.8 (Medium), based on vector [AV:A/AC:M/Au:N/C:C/I:C/A:C/E:F/RL:TF/RC:C](https://nvd.nist.gov/cvss.cfm?calculator&version=2&vector=\(AV:A/AC:M/Au:N/C:C/I:C/A:C/E:F/RL:TF/RC:C\)).
+The [CVSS](https://en.wikipedia.org/wiki/CVSS) score of this vulnerability is 6.8 (Medium), based on vector [AV:A/AC:M/Au:N/C:C/I:C/A:C/E:F/RL:TF/RC:C](https://nvd.nist.gov/cvss.cfm?calculator&amp;version=2&amp;vector=\(AV:A/AC:M/Au:N/C:C/I:C/A:C/E:F/RL:TF/RC:C\)).
```
Comment (patriknw) [inline akka-docs/src/main/paradox/security/2017-02-10-java-serialization.md]: Is this really correct? Existing link works both as [link in rendered html](https://doc.akka.io/docs/akka/current/security/2017-02-10-java-serialization.html) and when pasting into browser. The changed link doesn't work when pasting into browser. Do you mean that Paradox is replacing those `&amp;` with `&` when generating html?

**Review by patriknw — COMMENTED** [review]
what is the background to the `&amp;` change?

File: akka-docs/src/main/paradox/security/2017-02-10-java-serialization.md (hunk 2 above)
Comment (raboof) [reply akka-docs/src/main/paradox/security/2017-02-10-java-serialization.md]: Paradox is not replacing `&amp;` with `&`, but when encoding the link as the value of the `href` attribute it seems the [neat thing is to encode it](https://stackoverflow.com/questions/3705591/do-i-encode-ampersands-in-a-href) (even though modern browsers will indeed understand both).

I guess you could argue encoding those characters should be the responsibility of paradox, and not happen in the source document, though.

Comment (raboof) [reply akka-docs/src/main/paradox/security/2017-02-10-java-serialization.md]: The background-backgroud to this change is that I had written some Alpakka code that parsed these documents as XML, and the XML parser gets confused by them. But that was a one-off experiment anyway and I'm happy to drop this change if it's controversial ;)

Comment (patriknw) [reply akka-docs/src/main/paradox/security/2017-02-10-java-serialization.md]: ok

**Review by patriknw — APPROVED** [review]

**Comment by patriknw** [comment]
we can merge this, but it has a conflict now

**Pull request merged**